packages/react-native-editor/__device-tests__/gutenberg-editor-slash-inserter-@canary.test.js

```diff
@@ -25,7 +25,9 @@ async function assertSlashInserterPresent( checkIsVisible ) {
 	}
 }
 
-describe( 'Gutenberg Editor Slash Inserter tests', () => {
+// Due to flakiness, disabling until its more stable
```

Comment:
I wonder if we should add a link to an issue to track this. My thinking is, it'll be good to determine what's causing the flakiness, and re-enable, since we share parts of the implementation across platforms (so these tests are valuable to catch breakages before they're merged into trunk).

Reply:
Thanks for the suggestion! I agree, I added it here https://github.com/WordPress/gutenberg/pull/33197/commits/e3b3754e7117726ad199410e710125bee3acd7d3